seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -4584,7 +4584,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style[\\s\\S]*?color\\s*?:/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style\\s*?=\\s*[\\\"]{1}\\s*\\s*\\S*;*\\s*color\\s*?:\\s*?white\\s*;*\\s*[\\\"]{1}/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
Sorry for the messy email reply, not sure what happened there. I got as far as realizing that it would match either a backslash or a double quote in the end. What I realized after your comment is that because it ends up as `\"`, it doesn't match a backslash and a quote, but it's an escaped quote, so it just matches a quote. However, as far as I'm aware, quotes don't have to be escaped in a RegEx. I also don't understand why it's in a character set (the `[]`). http://codepen.io/Timoh/pen/vXqvZV?editors=0012